epochStart/metachain/economics_test.go

```diff
@@ -442,6 +443,161 @@ func TestEconomics_VerifyRewardsPerBlock_DifferentHitRates(t *testing.T) {
 	}
 }
 
+func TestComputeEndOfEpochEconomics(t *testing.T) {
+	t.Parallel()
+
+	commAddress := "communityAddress"
+	totalSupply, _ := big.NewInt(0).SetString("20000000000000000000000000000", 10) // 20 Billions ERD
+	nodePrice, _ := big.NewInt(0).SetString("1000000000000000000000", 10)          // 1000 ERD
+
+	roundDur := 4
+	args := getArguments()
+	args.RewardsHandler = &mock.RewardsHandlerStub{
+		MaxInflationRateCalled: func() float64 {
+			return 0.1
+		},
+		CommunityAddressCalled: func() string {
+			return commAddress
+		},
+		CommunityPercentageCalled: func() float64 {
+			return 0.1
+		},
+		LeaderPercentageCalled: func() float64 {
+			return 0.1
+		},
+	}
+	args.RoundTime = &mock.RoundTimeDurationHandler{
+		TimeDurationCalled: func() time.Duration {
+			return time.Duration(roundDur) * time.Second
+		},
+	}
```

Comment:
may create a function of setup.

Reply:
BTC dump ERD dump BTC pump ERD dump BTC stable ERD dump. Don'T buy ERD
  We want our money back, Romanian sons of bitches.
  
  Elrond was HACKED!
  Elrond announced that he had been hacked. Please don't buy ERD

Reply:
done.